<launch> 

  <arg name="nao_name"/>

  <node pkg="nao_pose_controller" type="nao_pose_controller" name="nao_pose_controller" output="screen">
    
    <param name="updating_frequency" value="120" type="double"/>

    <param name="x_tolerance"   value="0.07" type="double"/>
    <param name="y_tolerance"   value="0.07" type="double"/>
    <param name="yaw_tolerance" value="0.1"  type="double"/>

    <param name="next_goal_x_accept_treshold"   value="0.15" type="double"/>
    <param name="next_goal_y_accept_treshold"   value="0.15" type="double"/>
    <param name="next_goal_yaw_accept_treshold" value="0.2"  type="double"/>

    <param name="tf_polling_freq" value="30" type="str"/>

    <param name="global_frame_id" value="map" type="str"/>
    <param name="basefootprint_frame_id" value="base_footprint" type="str"/>

    <param name="robot_pose_time_th" value="1.0" type="double"/>

    <!-- If true position will be taken from base_footprint -->
    <param name="position_from_basefootprint" value="true" type="bool"/>

    <remap from="/marker/pose" to="$(arg nao_name)_marker/pose" />

  </node>

</launch>
